### ISSUE TITLE
[S7 Edge G935F] Call audio too loud on lowest Volume

### ISSUE THREAD

**Issue by supertiger1234** [phhusson/treble_experimentations#1346]
While in a call, the audio is too loud. Logs attached below.

[logall.log](https://github.com/phhusson/treble_experimentations/files/4722922/logall.log)
[media.log](https://github.com/phhusson/treble_experimentations/files/4722923/media.log)

**Comment by supertiger1234** [comment]
Yes, side note, there is even some crackling sounds when watching youtube video. Mainly the issue is with the call being too loud. What do you mean by "Media sound is scaled properly?" I don't understand 😅

**Comment by francob619** [comment]
I have the same problem with a moto e6 plus(helio p22)

**Comment by francob619** [comment]
I have the same problem with a moto e6 plus (helio p22)

**Comment by teichlch** [comment]
Same issue with the unihertz titan. (On earphones)
